opendbc/car/ford/values.py

```diff
@@ -68,7 +68,7 @@ class Footnote(Enum):
 
 @dataclass
 class FordCarDocs(CarDocs):
-  package: str = "Co-Pilot360 Assist+"
+  package: str = "A version of Co-Pilot360 which includes adaptive cruise control such as Co-Pilot360 Assist or Co-Pilot360 Active"
```

Comment:
```suggestion
  package: str = "Adaptive Cruise Control"
```

If `Co-Pilot360 Assist+` includes packages without ACC, what about this?

Reply:
Co-Pilot360 (No Assist/Active = No ACC)
Co-Pilot Assist = ACC and LCA
Co-Pilot Active = BlueCruise.

Technically, the current requirement is correct.  That hasn't stopped dozens of Maverick and Escape owners with regular Co-Pilot from buying a Comma and then complaining that the Comma website says it should be compatible.  I'm making it explicitly clear that they need to have ACC for the Comma to work, since the Co-Pilot naming convention is not very consistent or clear for Ford.  I'm open to other ways to try and make it clear that not all Co-Pilot versions include ACC.

Reply:
@alan-polk `Co-Pilot360 Assist = ACC but only LKA` - I think this isn't true for the Maverick 2023+, they use that name for the package with ACC and LCA. I think that's the only model with this difference though.

Adding a footnote to the Maverick seems like a good idea.